kit/preprocess.js

```diff
@@ -117,6 +117,45 @@ export const docstringPreprocess = {
 	}
 };
 
+// Preprocessor that converts markdown into FrameworkContent
+// svelte component using mdsvexPreprocess
+export const frameorkcontentPreprocess = {
+	markup: async ({ content }) => {
+		const REGEX_FRAMEWORKCONTENT = /<frameworkcontent>(((?!<frameworkcontent>).)*)<\/frameworkcontent>/gms;
+		const REGEX_PYTORCH = /<pytorch>(((?!<pytorch>).)*)<\/pytorch>/ms;
+		const REGEX_TENSORFLOW = /<tensorflow>(((?!<tensorflow>).)*)<\/tensorflow>/ms;
+		const REGEX_JAX = /<jax>(((?!<jax>).)*)<\/jax>/ms;
+		const FRAMEWORKS = [
+			{framework: "pytorch", REGEX_FW: REGEX_PYTORCH, isExist: false},
+			{framework: "tensorflow", REGEX_FW: REGEX_TENSORFLOW, isExist: false},
+			{framework: "jax", REGEX_FW: REGEX_JAX, isExist: false},
+		];
+
+		content = await replaceAsync(content, REGEX_FRAMEWORKCONTENT, async (_, docstringBody) => {
```

Comment:
```suggestion
		content = await replaceAsync(content, REGEX_FRAMEWORKCONTENT, async (_, frameworkBody) => {
```

Reply:
handled in https://github.com/huggingface/doc-builder/pull/130/commits/4fc7d0f3d3a0ad56fb3b05c5484fcf7d33082590